lib/gutenberg/gutenberg-editor-sidebar-component.js

```diff
@@ -242,7 +243,24 @@ export default class GutenbergEditorSidebarComponent extends AsyncBaseContainer
 		const trashSelector = By.css( 'button.editor-post-trash' );
 
 		await driverHelper.waitTillPresentAndDisplayed( this.driver, trashSelector );
-		return await driverHelper.clickWhenClickable( this.driver, trashSelector );
+		await driverHelper.clickWhenClickable( this.driver, trashSelector );
+
+		// wait for 'Move to trash' button to disappear
+		try {
+			return await driverHelper.waitTillNotPresent(
+				this.driver,
+				trashSelector,
+				this.explicitWaitMS * 3
+			);
+		} catch ( e ) {
+			// if it's still present send slack notification, post is deleted
+			return await SlackNotifier.warn(
```

Comment:
After adding `waitTillNotPresent()`, test `Calypso Gutenberg Editor: Schedule Basic Public Post @parallel` started to fail - "Move to trash" button didn't disappear, but post is deleted. This ensures that test doesn't fail when it comes to this step.